### DISCUSSION TITLE
Need Some Help With Theme Hero Image Settings

### DISCUSSION THREAD

**Discussion by nootkan** [GetPublii/Publii#1561] [Help]
I decided to tweak my hero image to only show on the homepage by deslecting the option button that controls this setting.  (see screenshot) Using the Simple theme v2.8.3.0.

However, now the hero image doesn't show at all and there is a huge blank space before the main content where it is supposed to be. The same holds true for all posts not just the homepage.

When I enable the show hero image on all pages, it is shown only on the homepage like I want but the other posts still have the large blank space where the hero image is supposed to be.  

How do I only show hero image on homepage while eliminating the blank space on all other posts where the hero image is supposed to be?

What am I missing?  See screenshots.

![ScreenHunter_640-Jun -29-09](https://github.com/GetPublii/Publii/assets/744769/03a4e53f-576d-4bf2-9c01-8d52d1df180c)

![homepage-screenshot-no-hero-image](https://github.com/GetPublii/Publii/assets/744769/5f7b7a74-d869-4f78-a875-0d8ea6667cc0)

![aboutpage-screenshot-no-hero-image](https://github.com/GetPublii/Publii/assets/744769/7f2b6ca4-02ad-4c5b-a632-86f540a3ad19)

![ScreenHunter_643-Jun -29-09 28](https://github.com/GetPublii/Publii/assets/744769/ae7bfa8e-e0bb-4548-87fc-93a89166d6fb)

![homepage-screenshot-hero-image](https://github.com/GetPublii/Publii/assets/744769/a7993409-37ac-4fd6-835e-d75d53015e80)

**Comment by bobmitro** [comment]
It's not a blank space; it's still a hero section with the post's title, metadata, and authors. It looks like a blank space in your case because you made it white. Just play with the CSS to customize it according to your needs.

**Comment by nootkan** [comment]
Thanks for your reply.  I realize that I could hide the hero image with css but that would eliminate it from the homepage also.  I just don't understand why the option to enable it for the homepage and no other page leaves the hero section on all the other pages/posts? Shouldn't it remove the code for the hero section from all other pages/posts and leave it on the homepage?

**Update:** I tried to remove the code myself from the post.hbs file (override) assuming that the same code would render for the homepage in front-page-post.hbs file but it did not. I did acheive what I wanted for all the posts (no hero image or blank space) but not the homepage (the hero image didn't show).

![ScreenHunter_646 Jun  30 11 06](https://github.com/GetPublii/Publii/assets/744769/3bc11044-fed5-45aa-8839-fae2a28a8600)
![ScreenHunter_647 Jun  30 11 06](https://github.com/GetPublii/Publii/assets/744769/254d5a38-9abd-469f-9385-50713e1a85a1)

**Comment by bobmitro** [comment]
There is no option to disable the header section. This is how the Simple theme works: the header with the title and metadata is displayed on a dark background, and its height can be adjusted via the theme settings. You can also upload a featured image that will cover the dark background.

**Comment by nootkan** [comment]
ok thanks. I guess I have to stay with Dreamweaver.  It was worth a shot.

**Reply by candidexmedia** [reply]
Hey @nootkan : I know you decided to stay with Dreamweaver, but if you're still interested in giving Publii another shot, I'd be open to taking a look if you share a live link (even just a quick dump on Netlify or GitHub pages) or your site files.
